### DISCUSSION TITLE
osmand can't get location

### DISCUSSION THREAD

**Discussion by bradrh1** [osmandapp/OsmAnd#13042] [Q&A]
I'm running osmand on a hammerhead karoo2 which runs android.  It can not determine the location.  Location permision is on.  LocusMaps works, osmand doesn't.   Any suggestions to get this to work?

**Comment by scaidermern** [comment]
OsmAnd requires Google PlayServices to obtain the location. Try OsmAnd from the F-Droid store, it doesn't has this requirement.
